admin/app/components/solidus_admin/adjustment_reasons/index/component.rb

```diff
@@ -47,8 +47,22 @@ def batch_actions
 
   def columns
     [
-      :name,
-      :code,
+      {
+        header: :name,
+        data: ->(adjustment_reason) do
+          link_to adjustment_reason.name, row_url(adjustment_reason),
```

Comment:
can we use `edit_path` instead of `row_url`? See https://github.com/solidusio/solidus/pull/6046/files#diff-0f1ecee6a27bb77a46f3b56a4af4b574b38755d675b6360039e6d3a82662aac4R28

Reply:
technically we can but we'll need to rewrite/remove behaviour for `rowClicked` handler in https://github.com/chaimann/solidus/blob/2e70faf61aeff21fcf702ec6a18c5e9d64489284/admin/app/components/solidus_admin/ui/table/component.js#L100, because without `row_url` it redirects you to root path if you click anywhere on the row except the link, so we'll have to wipe out the row click support entirely.

Maybe we could leave it there for now and address it in another PR

Reply:
Also fixed in #6046 8dca8490aedbdbc1840cfe1a52debf15ea8289b7